gruntfile.js

```diff
@@ -277,6 +285,8 @@ module.exports = function (grunt) {
   grunt.registerTask('test', ['env:test', 'lint', 'mkdir:upload', 'copy:localConfig', 'server', 'mochaTest', 'karma:unit']);
   grunt.registerTask('test:server', ['env:test', 'lint', 'server', 'mochaTest']);
   grunt.registerTask('test:client', ['env:test', 'lint', 'server', 'karma:unit']);
```

Comment:
I concur with @mleanos. For now, I will add e2e to the `test` task and keep the `test:e2e` separate from the `test:client` task.